pymc3/sampling.py

```diff
@@ -134,6 +151,20 @@ def sample(draws, step=None, init='ADVI', n_init=200000, start=None,
         in the system - 2.
     tune : int
         Number of iterations to tune, if applicable (defaults to None)
+    step_kwargs: dict
```

Comment:
How would we decide whether to interpret it as nested or not nested? I guess we could treat it as nested if the keys contain only valid step method names, but this doesn't sound clean. I'd prefer a separate `nuts_kwargs` argument. Or do you know a better way to decide?